### DISCUSSION TITLE
Mixture of samples

### DISCUSSION THREAD

**Discussion by AminaLEM** [GoekeLab/xpore#108] [Q&A] [answered]
Hello, 

I see in your paper that you performed a mixture of KO and WT, I would like to know how did you do that ? 

Thanks, 

Amina

**Comment by jonathangoeke** [comment, marked as answer]
Hi @AminaLEM,
The mixture was generated by extracting RNA from KO and WT Hek293T cells, and then combining the extracted RNA at specific ratios (25%, 50%, 75%). The combined RNA was then used for direct RNA sequencing. Does that answer your question?
Thanks
Jonathan

**Comment by AminaLEM** [comment]
Hi @jonathangoeke , 

Yes, Thank you. I was just confused about the method.

Thanks, 

Amina
